addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -515,15 +515,13 @@ var SnippetEditor = Widget.extend({
      * @param {boolean} show
      */
     toggleOptions: function (show) {
-        if (!this.$el) {
-            return;
-        }
-
-        if (this.areOptionsShown() === show) {
+        const makeToolbarVisible = checkEditorToolbarVisibility(this.options.wysiwyg.odooEditor.document);
+        if (!makeToolbarVisible && (!this.$el || this.areOptionsShown() === show)) {
             return;
         }
         this.trigger_up('update_customize_elements', {
             customize$Elements: show ? this._customize$Elements : [],
+            tab: makeToolbarVisible ? 'options' : undefined,
```

Comment:
Note: Normally `options` should be `this.tabs.OPTIONS` but we're in the wrong class for that...